uint16 mission_id
string mission_description
